charts/karavi-observability/templates/karavi-observability-configmap.yaml

```diff
@@ -65,3 +65,26 @@ data:
 
 {{ end }}
 
+---
+
+{{ if .Values.karaviMetricsPowerscale.enabled }}
+
+apiVersion: v1 
+kind: ConfigMap 
+metadata:
+  name: karavi-metrics-powerscale-configmap
+data:
+  karavi-metrics-powerscale.yaml : |
+    COLLECTOR_ADDR: {{ .Values.karaviMetricsPowerscale.collectorAddr }}
+    PROVISIONER_NAMES: {{ .Values.karaviMetricsPowerscale.provisionerNames }}
+    POWERSCALE_VOLUME_IO_POLL_FREQUENCY: "{{ .Values.karaviMetricsPowerscale.volumePollFrequencySeconds }}"
+    POWERSCALE_VOLUME_METRICS_ENABLED: "{{ .Values.karaviMetricsPowerscale.volumeMetricsEnabled }}"
```

Comment:
Not sure why PowerStore has a single VOLUME_METRICS_ENABLED to enable volume and cluster, but for PowerScale, we can have another `POWERSCALE_CLUSTER_METRICS_ENABLED`

Reply:
@baoy1  add clusterMetricsEnabled to values